components/pricing_modal/self_hosted_content.tsx

```diff
@@ -29,14 +30,32 @@ type ContentProps = {
     onHide: () => void;
 }
 
+const FALL_BACK_PROFESSIONAL_PRICE = '7.5';
+
 function SelfHostedContent(props: ContentProps) {
+    const [professionalPrice, setProfessionalPrice] = useState(' ');
```

Comment:
We could still initialize with the default price instead of an empty string. The idea was to catch the error so it wouldn't crash mostly

Reply:
Initially I had it like that, but in cases where the fall back price and the price set in stripe are different, we would get a flash of a wrong price